resources/teamcity/includes/tc-windows.inc.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+# Keyman is copyright (C) SIL Global. MIT License.
+
+download_symbol_server_index() {
+  # Download symbol server index from symbol server
+  builder_echo start "download symbol server index" "Downloading symbol server index"
+
+  (
+    # shellcheck disable=SC2164
+    cd "${KEYMAN_ROOT}/.."
+    # shellcheck disable=SC2154
+    powershell -NonInteractive -ExecutionPolicy Bypass -File "${THIS_SCRIPT_PATH}/../includes/download-symbol-server-index.ps1"
```

Comment:
`${THIS_SCRIPT_PATH}` is fragile because it is the calling script's path, which may vary in the future. Better to use `${KEYMAN_ROOT}/...`

```suggestion
    powershell -NonInteractive -ExecutionPolicy Bypass -File "${KEYMAN_ROOT}/resources/teamcity/includes/download-symbol-server-index.ps1"
```